test/utils/image/manifest.go

```diff
@@ -83,7 +83,7 @@ var (
 	NoSnatTestProxy          = ImageConfig{e2eRegistry, "no-snat-test-proxy", "1.0", true}
 	NWayHTTP                 = ImageConfig{e2eRegistry, "n-way-http", "1.0", true}
 	// When these values are updated, also update cmd/kubelet/app/options/options.go
-	Pause               = ImageConfig{gcRegistry, "pause", "3.0", false}
+	Pause               = ImageConfig{gcRegistry, "pause", "3.0", true}
```

Comment:
this should probably be `pause:3.1` now?

oh, or this is just being updated in #57517.  cc @verb